.github/workflows/deploy.yml

```diff
@@ -38,3 +38,18 @@ jobs:
         run: script/github-action-awscli-installation
       - name: Deploy the Serverless Demo App
         run: script/deploy-canary-demo
+  xcuitest:
+    runs-on: macos-11
```

Comment:
Do we need this? What is the default value?

Reply:
Yes, testing on ios 14 requires big sur